docs/build.sbt

```diff
@@ -25,14 +25,14 @@ FtpToFile / paradoxProperties ++= Map(
   "canonical.base_url" -> s"${homepage.value.get}/${FtpToFile.name}",
   "snip.build.base_dir" -> s"${baseDirectory.value}/../pekko-connectors-sample-${FtpToFile.name}",
   "github.root.base_dir" -> s"${baseDirectory.value}/..",
-  // Alpakka
-  "scaladoc.akka.stream.alpakka.base_url" -> s"https://doc.akka.io/api/alpakka/${Dependencies.Jms.AlpakkaVersion}",
-  "javadoc.akka.base_url" -> "",
-  "extref.alpakka.base_url" -> s"https://doc.akka.io/docs/alpakka/${Dependencies.FtpToFile.AlpakkaVersion}/%s",
-  // Akka
-  "scaladoc.akka.base_url" -> s"https://doc.akka.io/api/akka/${Dependencies.FtpToFile.AkkaVersion}",
-  "javadoc.akka.base_url" -> s"https://doc.akka.io/japi/akka/${Dependencies.FtpToFile.AkkaVersion}",
-  "extref.akka.base_url" -> s"https://doc.akka.io/docs/akka/${Dependencies.FtpToFile.AkkaVersion}/%s",
+  // Pekko Connectors
+  "scaladoc.pekko.connectors.base_url" -> s"https://pekko.apache.org/api/pekko-connectors/${Dependencies.FtpToFile.PekkoConnectorsVersion}/org/apache", // TODO: TBC
+  "javadoc.pekko.connectors.base_url" -> s"https://pekko.apache.org/japi/pekko-connectors/${Dependencies.FtpToFile.PekkoConnectorsVersion}/org/apache", // TODO: TBC
```

Comment:
I am wondering whether the version that snapshots use should be different to the actual version in the project. i.e. Pekko docs have a `current` alias which just points to the latest stable release (with projects like connectors which are still in snapshot its just latest snapshot but this will get fixed on first release).

Also pekko-connectors doesn't have any javadocs (i.e. japi) set up, that may be something that needs to be looked into.

Also is there a reason we replaced Jms to FtpFile?

Reply:
Also if you want the actual links to our documentation you can read https://pekko.apache.org/modules.html